### PR TITLE
[#175848009] Setup pipeline CD

### PR DESCRIPTION
**Pull request by balanza**
TBD

### CODE REVIEW

File: .devops/azure-templates/make-build.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    chmod +x env.sh && source env.sh
+    yarn build
+  env:
+    IO_DEVELOPER_PORTAL_PORT: $(IO_DEVELOPER_PORTAL_PORT)
+    IO_DEVELOPER_PORTAL_BACKEND: $(IO_DEVELOPER_PORTAL_BACKEND)
+    IO_DEVELOPER_PORTAL_APIM_BASE_URL: $(IO_DEVELOPER_PORTAL_APIM_BASE_URL)
+    IO_DEVELOPER_PORTAL_BASE_URL: $(IO_DEVELOPER_PORTAL_BASE_URL)
+    IO_DEVELOPER_PORTAL_PUBLIC_PATH: $(IO_DEVELOPER_PORTAL_PUBLIC_PATH)
+    IO_DEVELOPER_PORTAL_LOGO_PATH: $(IO_DEVELOPER_PORTAL_LOGO_PATH)
+  displayName: 'Build files'
```
Comment (balanza) [inline .devops/azure-templates/make-build.yml:20]: @GiovanniMancini please review these variables

File: .devops/deploy-pipelines.yml [hunk 2]
```diff
@@ -112,7 +112,7 @@ stages:
               azureSubscription: '$(STAGING_AZURE_SUBSCRIPTION)'
               destination: 'AzureBlob'
               storage: '$(STAGING_STORAGE_ACCOUNT_NAME)'
-              containerName: '$(BLOB_CONTAINER_NAME)'
+              containerName: 'xxx'
```
Comment (balanza) [inline .devops/deploy-pipelines.yml:115]: wat?

Comment (GiovanniMancini) [reply .devops/deploy-pipelines.yml:115]: I'm just trying to debug if I have a problem of naming or permission on the $web container :-)

**Pull request merged**